Source/WebKit/UIProcess/XR/openxr/OpenXRExtensions.h

```diff
@@ -23,6 +23,10 @@
 
 #include "OpenXRUtils.h"
 
+#if OS(ANDROID)
+#define jobject void*
+#endif
```

Comment:
What is this jobject?

Reply:
It's the type that the Java Native Interface (JNI) uses for handles to Java objects managed by the JVM.

Reply:
On second thoughts, on Android we can rely on `<jni.h>` always being available in the NDK, so I suppose it's not a big deal to include the header, guarder by `OS(ANDROID)`.